src/Microsoft.Azure.WebJobs.Extensions.Kafka/Trigger/KafkaTriggerAttributeBindingProvider.cs

```diff
@@ -49,7 +49,7 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
 
             var consumerConfig = CreateConsumerConfiguration(attribute);
 
-            var keyAndValueTypes = SerializationHelper.GetKeyAndValueTypes(attribute.AvroSchema, parameter.ParameterType, typeof(Ignore));
+            var keyAndValueTypes = SerializationHelper.GetKeyAndValueTypes(attribute.AvroSchema, parameter.ParameterType, typeof(string));
```

Comment:
@fbeltrao Do you know if we can specify the type as string for Avro use cases?

Reply:
This I tested and it's working fine for all Generic Avro cases